### ISSUE TITLE
Add option to use it as a plain picture viewer

### ISSUE THREAD

**Issue by Janneman84** [TimOliver/TOCropViewController#456]
I'm using the package for cropping but my app also needs descent plain picture viewer. So why not make this crop view double as a picture viewer?

You just have to hide some layers and buttons and the crop view becomes a good picture viewer!

I'm having a hard time finding a picture viewer that lets you rotate, like this VC, so hence the idea. I already did some hacking with KVO and view tree traversing to get it to work and it already works quite well!

I suggest you can set it with `CropViewCroppingStyle.none`

**Comment by knight2010** [comment]
Yep, It is good to implement a plain picture viewer, I just need one. Any update for that? So that we could use.

**Comment by TimOliver** [comment]
Thanks @Janneman84 and @knight2010!

Uhh, I'm not so sure about this. The whole goal of this view controller is to crop images with a very explicit in, and out, sort of implementation. Converting it to an image viewer feels like a sufficient deviation that it'll cause a lot more complexity.

Instead of converting this library into an image viewer, is there a reason you can't take another library (Like [this one](https://github.com/michaelhenry/ImageViewer.swift), or [this one](https://github.com/hyperoslo/Lightbox)) and just add image rotation capabilities to that one? :)

Thanks! I hope you understand where I'm coming from. :)

**Comment by Janneman84** [comment]
The other packages have no rotation and that is the tricky part... :(. I made something from scratch that kinda works, but it's tricky.

Since this package handles everything perfectly in my eyes it's just a matter of hiding a few layers and buttons (which I did with some hacking). I'm sure there is a little more to it than that, but how hard can it be? (haha)

I understand your point of single focus. However if it can be used as a picture viewer it can potentially attract a lot more users. Anyway, you call :).
